<?xml version="1.0"?>
<launch>

  <group ns="pioneer1">
	  <node name="my_node" pkg="laserscan_visualizer" type="try" output="screen">
	  	<remap from="/scan" to="/pioneer1/scan"/>
      <remap from="/camera/rgb/image_raw" to="/pioneer1/camera/rgb/image_raw"/>
	  </node>
  </group>

</launch>

